src/components/Theme/utils.ts

```diff
@@ -0,0 +1,25 @@
+import { kebabCase } from '~utils/string'
+import { UICustomisationOptions } from '~types/ui-customisation-options'
+
+export const setUICustomisations = (customUI: UICustomisationOptions): void => {
+  const sdkCustomisations = Object.keys(customUI).map(
+    (property: keyof typeof customUI) =>
+      `--osdk-${kebabCase(property)}: ${customUI[property]};`
+  )
```

Comment:
💅🏻  [nitpick]

```suggestion
  const sdkCustomisations = Object.entries(customUI).map(
    ([property, value]) => `--osdk-${kebabCase(property)}: ${value};`
  )
```

Reply:
Nice. I'm still a TS newbie so just went with this based off what I found when looking into fixing a type error.